packages/e2e-tests/specs/editor/blocks/gallery.test.js

```diff
@@ -44,7 +51,7 @@ describe( 'Gallery', () => {
 
 	it( 'can be created using uploaded images', async () => {
 		await insertBlock( 'Gallery' );
-		const filename = await upload( '.wp-block-gallery input[type="file"]' );
+		const filename = await placeholderUpload();
```

Comment:
Having switched to looking for any placeholders, what happens when there is more than one? Shouldn't `placeholderUpload` fail, for safety?

Reply:
Looking at the same pattern in the following tests, shouldn't we keep the specific selectors, like `.wp-block-gallery ...`?

Reply:
Sure we can keep it. If there's multiple gallery placeholders, it can still fail. I'll add the extra specificity.